### PR TITLE
Ticket issues resolved along with some ui fixes

### PR DESCRIPTION
**Pull request by ashifkhn**
Date: 03 02 2024

Developer Name: Ashif

---

## Description
Fixed goals flow, showing individuals goals, fixed golas form and date picker.
This includes a fixed goals form featuring essential fields like goal title, description, category, priority level, and an integrated date picker for selecting completion dates. Users can easily navigate through months and years to set deadlines. Each goal has a details section for comprehensive information, progress tracking, and notes. 

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [ ] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

</details>

https://github.com/Real-Dev-Squad/mobile-app/assets/54736284/a5ff10fb-a555-4a20-8bb3-5bda0dcae210

## Test Coverage
<details>
<summary>Screenshot 1</summary>

Screenshot 2024-02-03 at 2 18 27 PM
Screenshot 2024-02-03 at 2 18 19 PM

</details>

### CODE REVIEW

File: src/components/ToDoComponent/SettingGoals/CreateGoals.tsx [hunk 1]
```diff
@@ -212,83 +214,80 @@ const MainScreen = ({ navigation }) => {
...
   formView: {
-    borderWidth: 3,
-    paddingTop: 20,
-    paddingLeft: 30,
-    paddingRight: 30,
-    paddingBottom: 40,
-    height: 650,
-    borderRadius: 20,
-    overflow: 'hidden',
+    // borderWidth: 3,
```
Comment (shreya-mishra) [inline src/components/ToDoComponent/SettingGoals/CreateGoals.tsx]: can we remove these comments?

**Review by shreya-mishra — COMMENTED** [review]
Please remove Comments

**Comment by shreya-mishra** [comment]
> Date: 03 02 2024
> 
> Developer Name: Ashif
> 
> ## Issue Ticket Number
> ## Description
> Fixed goals flow, showing individuals goals, fixed golas form and date picker
> 
> ### Documentation Updated?
> * [ ]  Yes
> * [x]  No
> 
> ### Under Feature Flag
> * [ ]  Yes
> * [ ]  No
> 
> ### Database Changes
> * [ ]  Yes
> * [x]  No
> 
> ### Breaking Changes
> * [ ]  Yes
> * [x]  No
> 
> ### Development Tested?
> * [x]  Yes
> * [ ]  No
> 
> ## Screenshots
> Screenshot 1
> ## Test Coverage
> Screenshot 1
> ## Additional Notes

Please add issue description and proper video for the fixes

File: src/components/ToDoComponent/SettingGoals/CreateGoals.tsx [hunk 2]
```diff
@@ -158,7 +159,7 @@ const MainScreen = ({ navigation }) => {
           {isDropDownSelected ? (
             <View testID="user-container" style={styles.dropDownArea}>
               <TextInput
-                style={[styles.inputStyle, styles.searchBar]}
+                style={[styles.searchBar]}
```
Comment (shreya-mishra) [inline src/components/ToDoComponent/SettingGoals/CreateGoals.tsx]: this styling wont work

**Review by shreya-mishra — COMMENTED** [review]
I can still see the commented code

**Comment by ashifkhn** [comment]
> I can still see the commented code

removed

**Review by shreya-mishra — APPROVED** [review]

**Pull request merged**